tests/testthat/test-queue.R

```diff
@@ -58,3 +58,65 @@ test_that("Generated namespaced id if no ids exist", {
   q <- new_queue_quietly(root)
   expect_match(q$controller$queue_id, "orderly.runner")
 })
+
+
+test_that("Can submit task", {
+  skip_if_no_redis()
+
+  root <- test_prepare_orderly_example("data")
+  helper_add_git(root, c("src", "orderly_config.yml"))
+
+  q <- new_queue_quietly(root)
+  worker_manager <- start_queue_workers_quietly(1, q$controller)
+  make_worker_dirs(root, worker_manager$id)
```

Comment:
Could wrap this in a helper to get a queue with specified number of workers.

Reply:
done